### PR TITLE
NXCM-3737, cleanup left nav

### PR DESCRIPTION
**Pull request by bdemers**

### CODE REVIEW

**Comment by bdemers** [comment]
and one small change in the staging plugin: https://github.com/sonatype/nexus-enterprise/pull/114

File: nexus/nexus-webapp/src/main/webapp/js/extensions/Sonatype.navigation.js [hunk 1]
```diff
@@ -258,6 +263,33 @@ Ext.extend(Sonatype.navigation.Section, Ext.Panel, {
...
         }
         return Sonatype.navigation.Section.superclass.add.call(this, c);
+      },
+
+      sort : function(asOrder) {
+        if(!this.items)
+        {
+            return;
+        }
+
```
Comment (bdemers) [inline nexus/nexus-webapp/src/main/webapp/js/extensions/Sonatype.navigation.js:273]: Do we have some util or extjs function to handle this?  I really should not need to do this...

File: nexus/nexus-webapp/src/main/webapp/js/extensions/Sonatype.navigation.js [hunk 2]
```diff
@@ -121,6 +123,7 @@ Sonatype.navigation.Section = function(config) {
   var config = config || {};
   var defaultConfig = {
     collapsible : true,
+    titleCollapse: true,
```
Comment (bdemers) [inline nexus/nexus-webapp/src/main/webapp/js/extensions/Sonatype.navigation.js:126]: @peterlynch, this line if for you

Comment (peterlynch) [inline nexus/nexus-webapp/src/main/webapp/js/extensions/Sonatype.navigation.js:126]: :beer: *12
now we can call it 2.0!

**Comment by peterlynch** [comment]
+1

**Comment by velo** [comment]
+1

**Comment by nabcos** [comment]
+1

**Comment by adreghiciu** [comment]
+1

File: nexus/nexus-webapp/src/main/webapp/js/extensions/Sonatype.navigation.js (hunk 1 above)
Comment (dbradicich) [inline nexus/nexus-webapp/src/main/webapp/js/extensions/Sonatype.navigation.js:273]: No, best you can do is streamline the sort function a bit (and maybe add sort to Sonatype.utils that takes a sort function)

var sortFn = function(r1, r2){
    var v1 = Sonatype.utils.lowercase(r1), v2 = Sonatype.utils.lowercase(r2);
    return v1 > v2 ? 1 : (v1 < v2 ? -1 : 0);
};

this.items.sort(asOrder || 'ASC', sortFn);

Comment (bdemers) [inline nexus/nexus-webapp/src/main/webapp/js/extensions/Sonatype.navigation.js:273]: Thanks, that is better then the below

**Comment by mcculls** [comment]
+1

File: nexus/nexus-webapp/src/main/webapp/js/extensions/Sonatype.navigation.js (hunk 1 above)
Comment (dbradicich) [inline nexus/nexus-webapp/src/main/webapp/js/extensions/Sonatype.navigation.js:273]: +1

**Pull request merged**